sdk/resources/Microsoft.Azure.Management.Resource/src/generate.ps1

```diff
@@ -3,7 +3,10 @@
 # and copy these new set of files to the main Generated folder. This way exisiting files will not be deleted.
 
 # Generate package with resources tag
-Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-resources-2020-10" -SdkGenerationDirectory "$PSScriptRoot\Generated\Resources"
+#Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-resources-2020-10" -SdkGenerationDirectory "$PSScriptRoot\Generated\Resources"
+
+# Generate package with templatespecs tag
+Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-templatespecs-2021-03-preview" -SdkGenerationDirectory "$PSScriptRoot\Generated\TemplateSpecs"
```

Comment:
it is highly recommended to keep this file simple and keep the package version in the default package tag in the readme.md in swagger repo. See other RP's simple generate.ps1 for examples.

Reply:
I can push a commit to revert this file to the way it was before this PR, if that works. 
As for a default tag, from what I know, the resources/resource-manager swagger spec has multiple tags to separate some components and I don't know if a default tag exists.

Reply:
You don't have to revert. It is just less work. the default tag is just the `tag:` in the readme.md. Should be only one there. Anyway, I merged for you. It's a recommendation to potentially avoid mistakes.